### DISCUSSION TITLE
where's the images gallery?

### DISCUSSION THREAD

**Discussion by tybiboune** [lllyasviel/stable-diffusion-webui-forge#328] [Q&A]
Sorry for what is very probably a stupid noob question, but I can't seem to find the "images gallery" as it used to be in its own tab in A1111. Could someone point me at it please? Thank you in advance :)

**Comment by Moussack** [comment]
I use this, it's even better than A1111's. Works pretty well with forge
[https://github.com/zanllp/sd-webui-infinite-image-browsing](https://github.com/zanllp/sd-webui-infinite-image-browsing)

**Reply by ophoria** [reply]
+1 for that, it's the best imho....

**Reply by tybiboune** [reply]
brilliant!!!! thanks for that!

**Reply by Irmagon** [reply]
Try send image to txt2img )))
Cause error:
```
  File "Z:\webui_forge\webui\modules\styles.py", line 45, in extract_style_text_from_prompt
    left, right = stripped_style_text.split("{prompt}", 2)
ValueError: too many values to unpack (expected 2)
```

**Comment by freecoderwaifu** [comment]
It's been a standalone extension for a long while, more than a year at this point. Search "gallery" or "browser" in the Extensions > Available tab, there are also more alternatives like the one already posted.
